src/components/Api.js

```diff
@@ -0,0 +1,168 @@
+class Api {
+  constructor(options) {
+    this._authorization = "fd5203a5-1f57-4428-b210-42e565daa250";
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: this._authorization,
+      },
+    })
+      .then((res) => {
```

Comment:
Needs correction: The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```